tests/XsdDateDecoder.spec.js

```diff
@@ -0,0 +1,72 @@
+/*!
+* Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
+*/
+// import {
+//   expect
+// } from 'chai';
+// import {XsdDateDecoder} from '../lib/codecs/XsdDateDecoder.js';
+
+describe.skip('XsdDateDecoder', () => {
+  /*
```

Comment:
Why is this being skipped?

Reply:
These are old tests that need to be updated. I left them as an exercise for someone else who is going to take over writing tests to have something to do to get started. There are some other round trip tests that use xsd datetimes in the main suite.